ggml.c

```diff
@@ -11299,7 +11391,8 @@ static void ggml_compute_forward_mul_mat(
     if (params->type == GGML_TASK_INIT) {
         if (src1->type != vec_dot_type) {
             char * wdata = params->wdata;
-            const size_t row_size = ne10*ggml_type_size(vec_dot_type)/ggml_blck_size(vec_dot_type);
+            const size_t row_size = ggml_type_size(vec_dot_type)*(ne10 + ggml_blck_size(vec_dot_type) - 1)
+                / ggml_blck_size(vec_dot_type);
```

Comment:
So presumably this is then the change from my end that is causing the issue. But if this change makes a difference that would imply that `ne10` is not a multiple of `ggml_blck_size(vec_dot_type)`.